common/src/main/java/com/thekillerbunny/goofyplugin/GoofyPlugin.java

```diff
@@ -25,6 +27,8 @@ public class GoofyPlugin implements FiguraAPI {
     public static final Logger LOGGER = LoggerFactory.getLogger("GoofyPlugin");
     private Avatar avatar;
 
+    public static HashMap<String, Boolean> disabledElements = new HashMap<String, Boolean>();
```

Comment:
this is static, meaning it will be shared between *all avatars*, but ig checking for local makes it fine

Reply:
it only will affect local, and I made it static so I can easily check values